btc-withdrawal/src/main/kotlin/withdrawal/btc/statistics/WithdrawalStatistics.kt

```diff
@@ -0,0 +1,22 @@
+package withdrawal.btc.statistics
+
+import java.util.concurrent.atomic.AtomicInteger
+
+/**
+ * Data class that holds short statistics about withdrawal service
+ */
+data class WithdrawalStatistics(
+    val totalTransfers: AtomicInteger,
+    val failedTransfers: AtomicInteger,
+    val succeededTransfers: AtomicInteger
+) {
+    fun incTotalTransfers() = totalTransfers.incrementAndGet()
```

Comment:
Do we need to keep a total value which is always a sum of failed and succeeded? Or not?

Reply:
In my opinion, we need this. For example, I don't want to do any math. I just want to see numbers.

Reply:
Time vs Space holy war confirmed :) So do we need to estimate how often we update the total value and determine which overhead is more significant?

Reply:
I don't think the overhead is big, so we can leave the value as it is